libs/category/src/models/category-breadcrumb.ts

```diff
@@ -7,7 +7,17 @@ import { DaffCategory } from './category';
  * describing the traversal to a particular category within a tree of categories.
  */
 export interface DaffCategoryBreadcrumb extends DaffLocatable {
+	/**
+	 * The id of the associated category.
+	 */
   id: DaffCategory['id'];
+	/**
+	 * A label to display the category name in the UI.
+	 */
   name: DaffCategory['name'];
+	/**
+	 * The number of nodes between this node and the root node for the category trie. For example, the level for a
```

Comment:
is `trie` supposed to be `tree`?

Reply:
I meant `trie` but I think I'm misusing the term. I'll switch to "tree"

Reply:
changed